src/platform/silabs/wifi/rs911x/WifiInterface.cpp

```diff
@@ -599,33 +599,33 @@ void HandleDHCPPolling(void)
      */
     if ((ip6_addr_ispreferred(netif_ip6_addr_state(sta_netif, 0))) && !hasNotifiedIPV6)
     {
-        wfx_ipv6_notify(GET_IPV6_SUCCESS);
-        hasNotifiedIPV6 = true;
-        WifiEvent event = WifiEvent::kStationDhcpDone;
+        NotifyIPv6Change(true);
```

Comment:
must NotifyConnectivity and NotifyIPv6Change be treated separately?

Reply:
They don't but i'm keeping that for a follow up PR. The goal is to first clean up in the interface files then to clean the platform implementations.